player/Player.js

```diff
@@ -0,0 +1,10 @@
+export default class Player {
+  constructor() {
+    this.name = 'Robert';
+    this.hp = 100;
+    this.atk = 20;
+    this.def = 7;
+    this.skills;
```

Comment:
This line does nothing without assigning something
```suggestion
    this.skills = [];
```